frontend/dev.Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:alpine
+
+LABEL version="0.1"
+LABEL project="meep"
+LABEL component="frontend"
+
+RUN npm install -g --no-progress --ignore-optional webpack webpack-cli
+
+WORKDIR /tmp
+COPY package.json /tmp/
+RUN npm install --no-progress --ignore-optional
+
+WORKDIR /usr/src/app
```

Comment:
Is it necessary to move `WORKDIR` around?

Reply:
It depends on the subsequent actions.  In this case, I want to install the node packages in one place and then change the working directory for the app code.